src/games/strategy/triplea/attachments/TerritoryAttachment.java

```diff
@@ -138,15 +138,7 @@ public static Territory getFirstOwnedCapitalOrFirstUnownedCapital(final PlayerID
    * Convenience method. Can return null.
    */
   public static TerritoryAttachment get(final Territory t) {
-    return get(t, false);
-  }
-
-  public static TerritoryAttachment get(final Territory t, final boolean allowNull) {
-    final TerritoryAttachment rVal = (TerritoryAttachment) t.getAttachment(Constants.TERRITORY_ATTACHMENT_NAME);
-    if (!allowNull && rVal == null && !t.isWater()) {
-      throw new IllegalStateException("No territory attachment for:" + t.getName());
-    }
-    return rVal;
+    return (TerritoryAttachment) t.getAttachment(Constants.TERRITORY_ATTACHMENT_NAME);
```

Comment:
Doesn't this change functionality for null territories since we are passing false?